src/Compilers/CSharp/Test/Syntax/Parsing/AnonymousFunctionParsingTests.cs

```diff
@@ -1476,13 +1476,12 @@ public void StaticLambdaArrayInitializer_Incomplete_01(bool useCsharp9)
             var test = @"_ = new Action[] { static }";
 
             UsingStatement(test, options: useCsharp9 ? TestOptions.Regular9 : TestOptions.Regular8,
-                // (1,20): error CS1003: Syntax error, ',' expected
+                // (1,20): error CS1041: Identifier expected; 'static' is a keyword
                 // _ = new Action[] { static }
-                Diagnostic(ErrorCode.ERR_SyntaxError, "static").WithArguments(",").WithLocation(1, 20),
+                Diagnostic(ErrorCode.ERR_IdentifierExpectedKW, "static").WithArguments("", "static").WithLocation(1, 20),
```

Comment:
slight change from "syntax error" to "identifier expected" (the latter being what we report uniformly for all other separated list parsing.